kubernetes/samples/scripts/create-rcu-schema/stop-db-service.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+# Drop the DB Service created by start-db-service.sh
+
+script="${BASH_SOURCE[0]}"
+scriptDir="$( cd "$( dirname "${script}" )" && pwd )"
+
+dbpod=`kubectl get po | grep oracle-db | cut -f1 -d " " `
+kubectl delete -f ${scriptDir}/common/oradb.yaml --ignore-not-found
```

Comment:
the script is called "stop" but you are deleting - seems wrong - rename script?

Reply:
Here I stop the service by internally deleting the kubernate service. Start/Stop Service also complements each other with reverse action.